ee/tabby-webserver/src/service/thread.rs

```diff
@@ -477,6 +482,26 @@ mod tests {
         assert!(non_existent_thread.is_none());
     }
 
+    #[tokio::test]
+    async fn test_delete_thread() {
```

Comment:
please also verify messages of the thread got deleted as part of cascading deletion

https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/a7d29b822c7cdcad03873c308412656a727809da/ee/tabby-db/schema/schema.sql#L185